### PR TITLE
Fix LocalPubSubSpec instability

### LINKED ISSUE #30466: LocalPubSubSpec: must publish to all local subscriber actors of a topic
https://github.com/akka/akka/runs/3239867889?check_suite_focus=true

```
must publish to all local subscriber actors of a topic *** FAILED *** (3 seconds, 117 milliseconds)
[info]   java.lang.AssertionError: timeout (3 seconds) during expectMessage while waiting for banana
[info]   at akka.actor.testkit.typed.internal.TestProbeImpl.assertFail(TestProbeImpl.scala:399)
[info]   at akka.actor.testkit.typed.internal.TestProbeImpl.expectMessage_internal(TestProbeImpl.scala:171)
[info]   at akka.actor.testkit.typed.internal.TestProbeImpl.expectMessage(TestProbeImpl.scala:149)
[info]   at akka.actor.typed.pubsub.LocalPubSubSpec.f$proxy1$2(LocalPubSubSpec.scala:43)
[info]   at akka.actor.typed.pubsub.LocalPubSubSpec.$init$$$anonfun$2$$anonfun$1(LocalPubSubSpec.scala:20)
```

### PR DESCRIPTION
**Pull request by raboof**
Wait for the topic to be created

Fixes #30466

### CODE REVIEW

File: akka-actor-typed-tests/src/test/scala/akka/actor/typed/pubsub/LocalPubSubSpec.scala [hunk 1]
```diff
@@ -36,7 +36,9 @@ class LocalPubSubSpec extends ScalaTestWithActorTestKit with AnyWordSpecLike wit
         val statsProbe = testKit.createTestProbe[TopicImpl.TopicStats]()
         statsProbe.awaitAssert {
           fruitTopic ! TopicImpl.GetTopicStats(statsProbe.ref)
-          statsProbe.receiveMessage().localSubscriberCount should ===(3)
+          val stats = statsProbe.receiveMessage()
```
Comment (andreaTP) [inline akka-actor-typed-tests/src/test/scala/akka/actor/typed/pubsub/LocalPubSubSpec.scala:39]: how this is waiting?
the code is equivalent if I understand it correctly

Comment (raboof) [reply akka-actor-typed-tests/src/test/scala/akka/actor/typed/pubsub/LocalPubSubSpec.scala:39]: `awaitAssert` runs the block until it succeeds (or times out).

Previously it only waited for `localSubscriberCount should ===(3)`, now it also waits for `topicInstanceCount should ===(1)`

Comment (andreaTP) [reply akka-actor-typed-tests/src/test/scala/akka/actor/typed/pubsub/LocalPubSubSpec.scala:39]: I see!

**Review by andreaTP — APPROVED** [review]

**Comment by andreaTP** [comment]
LGTM

**Pull request merged**